policies/coding-style.md

```diff
@@ -464,6 +464,11 @@ derived types. To this end, use just one data declaration per line (no
 commas for multiple data declarations). This leaves you room for a small
 comment on each item, explaining its use.
 
+In an effort to better translate our source code into documentation that is more
+easily understandable to future developers, please also consider for adding
+Doxygen style comments to any function/data structures/macros/etc that you alter
+or create in the development of patches for OpenSSL.
```

Comment:
Does this apply to internal / static things too?  I assume this is mostly to be done in header files...

Reply:
I wouldn't expect so.  Doxygen is meant for users not developers.
That doesn't mean we cannot document our internals, however I believe that doc/internal is a better place.

Reply:
So the point with what I said is that maybe it should be specified somewhat clearly, something along the lines of "This is expected to happen primarly for ... and primarly in ...".  This is a policy after all.

Reply:
@levitte honestly I think thats up for debate.  @vavroch2010 and I were discussing this as a means for making all of our code a bit more accessible to new developers, so I could see starting to document all our code in this manner as beneficial.  I just thought I would propose it as a means to start debate.  That said, @paulidale is correct.  Typically, Doxygen is used for public apis, though thats in no way a requirement